tgui/packages/tgui/interfaces/SecurityRecords.jsx

```diff
@@ -0,0 +1,688 @@
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{
+                  __html: record.security_incident || 'None',
+                }}
+              />
+            </Box>
+            <hr />
+            <Box textAlign="center" style={sectionHeaderStyle}>
+              Comments Log
+            </Box>
+            <Box style={boxStyle}>
+              {record.security_comments &&
+              Object.keys(record.security_comments).length > 0
+                ? Object.entries(record.security_comments).map(
+                    ([key, comment]) => (
+                      <Box
+                        key={key}
+                        style={{ marginBottom: '10px', padding: '5px' }}
+                      >
+                        {comment.deleted_by ? (
+                          <Box style={grayItalicStyle}>
+                            Comment deleted by {comment.deleted_by} at{' '}
+                            {comment.deleted_at || 'unknown time'}.
+                          </Box>
+                        ) : (
+                          <>
+                            <Box fontSize="1.2rem">{comment.entry}</Box>
+                            <Box style={{ fontSize: '0.9rem', color: 'gray' }}>
+                              Created at: {comment.created_at} /{' '}
+                              {comment?.created_by?.name} (
+                              {comment?.created_by?.rank}){' '}
+                            </Box>
+                            <Button
+                              onClick={() => {
+                                act('delete_comment', {
+                                  id: selectedRecord.id,
+                                  key,
+                                });
+                              }}
+                              mt={1}
+                            >
+                              Delete
+                            </Button>
+                          </>
+                        )}
+                      </Box>
+                    ),
+                  )
+                : 'No comments available.'}
+            </Box>
+            <Box style={{ ...boxStyle, paddingLeft: '2px' }}>
+              <Button onClick={() => setCommentModalOpen(true)}>
+                Add Comment
+              </Button>
+            </Box>
+          </>
+        )}
+
+        <hr />
+        <Flex direction="row" gap={2}>
+          <Button
+            onClick={() => act('print_personal_record', { id: record.id })}
+            color="blue"
+          >
+            Print record
+          </Button>
+          <Button.Confirm
+            fluid
+            color="red"
+            confirmColor="bad"
+            confirmContent="Confirm?"
+            onClick={() => act('delete_general_record', { id: record.id })}
+          >
+            Delete general record
+          </Button.Confirm>
+        </Flex>
+
+        <hr />
```

Comment:
`<Divider>` component

Reply:
Changed all <hr/> elements to <Divider/>